### PR TITLE
SCP-2021 - Add metadata to simulator

### PR DESCRIPTION
**Pull request by palas**
This PR displays the metadata in the simulator and instantiation form:
- Adds explanation to the places where it is relevant
- Adds explanation to the hints (abbr) that already existed for roles
- Fixes several glitches in the CSS of the simulator (in order to make it more responsive)

metadata-demo

Deployed to https://pablo.marlowe.iohkdev.io/

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [x] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [x] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [x] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [x] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [x] Someone approved it
- [x] Commits have useful messages
- [x] Review clarifications made it into the code
- [x] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: marlowe-playground-client/static/css/panels.scss [hunk 1]
```diff
@@ -28,6 +28,10 @@
   align-self: flex-start;
 }
 
+.align-center {
```
Comment (hrajchert) [inline marlowe-playground-client/static/css/panels.scss:31]: If we are going to use tailwind convention for the CSS utilities this could be called `self-center` 

https://tailwindcss.com/docs/align-self

File: marlowe-playground-client/static/css/panels.scss [hunk 2]
```diff
@@ -208,6 +214,55 @@ button.minus-btn:hover {
   margin-bottom: 0.5rem;
 }
 
+.action-group {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.action-group-explanation {
+  margin-left: 2rem;
```
Comment (hrajchert) [inline marlowe-playground-client/static/css/panels.scss:223]: Try not to use absolute measures and use the `var(--medium-margin);` or `var(--small-margin);` instead.

File: marlowe-playground-client/static/css/panels.scss [hunk 3]
```diff
@@ -208,6 +214,55 @@ button.minus-btn:hover {
...
+.slot-input {
+  display: flex;
+  flex-wrap: wrap;
+  line-height: normal;
+}
+
+.action-input {
+  flex-grow: 1;
+  margin-left: 0.5rem;
+  min-width: 4rem;
```
Comment (hrajchert) [inline marlowe-playground-client/static/css/panels.scss:262]: why did you need to set this min-width?

**Review by hrajchert — APPROVED** [review]
LGTM... minor comments, try to use `var` in CSS instead of absolute values. Also, I think I'd prefer to have metadata inside of `"double quotes"` rather than `“high and low„`

File: marlowe-playground-client/static/css/panels.scss (hunk 3 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:262]: So that the explanation wraps around if the input is too small

Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:262]: Ah, I guess I could also have set flex-shrink to 0 and a width instead, but well

File: marlowe-playground-client/static/css/panels.scss (hunk 1 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:31]: I will have it in mind in the future. For this one in particular `align-top` was already there, so it is probably best to have them consistent
